example/lib/custom_marker.dart

```diff
@@ -0,0 +1,243 @@
+            for (var i = 0; i < RANDOM_MARKER_NUM; i++) {
+              var point = Point<double>(value[i].x, value[i].y);
+              _addMarker(point, param[i]);
+            }
+          });
+        },
+        child: Icon(Icons.add),
+      ),
+    );
+  }
+
+  void _measurePerformance() {
+    final trial = 10;
+    final batches = [500, 1000, 1500, 2000, 2500, 3000];
+    var results = Map<int, List<double>>();
+    for (final batch in batches) {
+      results[batch] = [0.0, 0.0];
+    }
+
+    mapController.toScreenLocation(LatLng(0, 0));
+    Stopwatch sw = Stopwatch();
+
+    for (final batch in batches) {
+      //
+      // primitive
+      //
+      for (var i = 0; i < trial; i++) {
+        sw.start();
+        var list = <Future<Point<num>>>[];
+        for (var j = 0; j < batch; j++) {
+          var p = mapController.toScreenLocation(LatLng(j.toDouble() % 80, j.toDouble() % 300));
+          list.add(p);
+        }
+        Future.wait(list);
+        sw.stop();
+        results[batch][0] += sw.elapsedMilliseconds;
+        sw.reset();
+      }
+
+      //
+      // batch
+      //
+      for (var i = 0; i < trial; i++) {
+        sw.start();
+        var param = <LatLng>[];
+        for (var j = 0; j < batch; j++) {
+          param.add(LatLng(j.toDouble() % 80, j.toDouble() % 300));
+        }
+        Future.wait([mapController.toScreenLocationBatch(param)]);
+        sw.stop();
+        results[batch][1] += sw.elapsedMilliseconds;
+        sw.reset();
+      }
+
+      print('batch=${batch},primitive=${results[batch][0] / trial}ms, batch=${results[batch][1] / trial}ms');
+    }
+
+  }
+}
+
+class Marker extends StatefulWidget {
+  Point _position;
+  set Position(Point point) { _position = point; _state.Position = point;}
+  LatLng _coordinate;
+  LatLng get Coordinate => _coordinate;
+  _MarkerState _state;
+
+  Marker(String key, this._position, this._coordinate) : super(key: Key(key));
+
+  @override
+  State<StatefulWidget> createState() {
+    _state = _MarkerState(_position);
+    return _state;
+  }
+}
+
+class _MarkerState extends State with TickerProviderStateMixin {
+  final _iconSize = 20.0;
+
+  Point _position;
+  set Position(Point point) {
+    setState(() {
+      _position = point;
+    });
+  }
+
+  AnimationController _controller;
+  Animation<double> _animation;
+
+  _MarkerState(this._position);
+
+  @override
+  void initState() {
+    super.initState();
+    _controller = AnimationController(
+      duration: const Duration(seconds: 2),
+      vsync: this,
+    )..repeat(reverse: true);
+    _animation = CurvedAnimation(
+      parent: _controller,
+      curve: Curves.elasticOut,
+    );
+  }
+
+  @override
+  void dispose() {
+    _controller.dispose();
+    super.dispose();
+  }
+
+  @override
+  Widget build(BuildContext context) {
+    var ratio = 1.0;
+
+    try {
+      ratio = Platform.isIOS ? 1.0 : MediaQuery.of(context).devicePixelRatio; //TODO: iOS returns lp
```

Comment:
Android returns screen pixel while iOS returns logical pixel. The difference comes from the behavior of underlying SDKs.

Reply:
Is it ok to remove the todo then?

Reply:
This is good to document in a comment, but no need for a todo.

Also, it would be better to wrap it in a `if (!kIsWeb)` block rather than `try-catch` as that is what is expected.